code/modules/admin/verbs/adminjump.dm

```diff
@@ -152,6 +152,7 @@
 	var/startdensity = mover.density
 	var/startflags = mover.pass_flags
 	var/startincorporeal = 0
+	var/startdestinationdensity = newloc.density
 	if(istype(mover, /mob/living))
 		var/mob/living/L = mover
 		startincorporeal = L.incorporeal_move

```

Comment:
Remove everything related to _incorporeal_move_, it's only checked in Client/Move(), hence useless here.